content/en/blog/_posts/2024-04-xx-structured-authz-beta.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: blog
+title: 'Multi-Webhook Authorization Made Easy'
+date: 2024-04-nn
+slug: multi-webhook-authorization-made-easy
```

Comment:
Thinking more about the title. Should we instead focus on "Authorization made more modular and powerful"?

The feature did start with the thought of adding multiple authz webhooks support but we achieved much more in the process.

Reply:
sounds good. 
how about `Enhancing Security: Simplified Multi-Webhook and Modular Authorization`
I would like to keep multi-webhook if possible since that is the main pain point and motivation driving this enhancement